### PR TITLE
Added mysql over ssl connection support

### PR DESCRIPTION
**Pull request by TheBlueKingLP**
Added option to secure database connection with SSL

### CODE REVIEW

**Review by bastianleicht — APPROVED** [review]

File: config/database.php [hunk 1]
```diff
@@ -45,7 +45,14 @@
             'collation' => 'utf8mb4_unicode_ci',
             'prefix' => env('DB_PREFIX', ''),
             'strict' => env('DB_STRICT_MODE', false),
+            'sslmode' => env('DB_SSLMODE', 'prefer'),
             'timezone' => env('DB_TIMEZONE', Time::getMySQLTimezoneOffset(env('APP_TIMEZONE', 'UTC'))),
+            'options'   => array(
+                PDO::MYSQL_ATTR_SSL_VERIFY_SERVER_CERT => env('DB_VERIFY_SERVER_CERT', false),
```
Comment (DaneEveritt) [inline config/database.php:51]: This should be defaulted true and people can set it to false if they explicitly need it disabled.

File: config/database.php [hunk 2]
```diff
@@ -45,7 +45,14 @@
...
+                PDO::MYSQL_ATTR_SSL_KEY => env('DB_SSL_KEY', ''),
+                PDO::MYSQL_ATTR_SSL_CERT => env('DB_SSL_CERT', ''),
+                PDO::MYSQL_ATTR_SSL_CA => env('DB_SSL_CA', ''),
```
Comment (DaneEveritt) [inline config/database.php:54]: These environment names should all be named following the constant name, e.g. `MYSQL_ATTR_SSL_CA`, similar to [Laravel's default setup](https://github.com/laravel/laravel/blob/8.x/config/database.php#L61-L63).

File: config/database.php [hunk 3]
```diff
@@ -45,7 +45,14 @@
             'collation' => 'utf8mb4_unicode_ci',
             'prefix' => env('DB_PREFIX', ''),
             'strict' => env('DB_STRICT_MODE', false),
+            'sslmode' => env('DB_SSLMODE', 'prefer'),
             'timezone' => env('DB_TIMEZONE', Time::getMySQLTimezoneOffset(env('APP_TIMEZONE', 'UTC'))),
+            'options'   => array(
```
Comment (DaneEveritt) [inline config/database.php:50]: This codebase does not use `array()`, you should be using `[]` and there should only be a single space between the key and the arrow.

File: .env.example [hunk 4]
```diff
@@ -14,6 +14,11 @@ DB_PORT=3306
 DB_DATABASE=panel
 DB_USERNAME=pterodactyl
 DB_PASSWORD=
+DB_SSLMODE=
```
Comment (DaneEveritt) [inline .env.example:17]: These don't need to exist in the example file.

**Review by DaneEveritt — CHANGES_REQUESTED** [review]

**Comment by matthewpi** [comment]
Replaced by https://github.com/pterodactyl/panel/pull/3464

**Pull request closed without merging**